Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SimulatedDensityOfStates.py

```diff
@@ -488,7 +491,7 @@ def _compute_partial_ion_workflow(self, partial_ions, frequencies, eigenvectors,
         total_workspace = None
 
         # Output each contribution to it's own workspace
-        for ion_name, ions in partial_ions.items():
+        for ion_name, ions in list(partial_ions.items()):
```

Comment:
You don't need to transform to list when you iterate over it. There are a fewer other places in this pull request where 2to3 did this also.